ichnaea/config.py

```diff
@@ -20,15 +20,15 @@
 with open(CONTRIBUTE_FILE, 'r') as fd:
     CONTRIBUTE_INFO = simplejson.load(fd)
 
-VERSION = pkg_resources.get_distribution('ichnaea').version
```

Comment:
Since we no longer install ichnaea, we can't pull the version from the distribution data.